e2e-tests/portal/cypress/e2e/journeys/maker/dashboard/deals/dashboard-table-data.spec.js

```diff
@@ -97,17 +96,15 @@ context('View dashboard deals as a maker', () => {
     //---------------------------------------------------------------
     // second deal (BSS)
     //---------------------------------------------------------------
-    cy.get('table tr').eq(2).find(`[data-cy="deal__status--${bssDealId}"]`).should('exist');
-
-    cy.assertText(exporter(bssDealId), bssDeal.exporter.companyName);
+    cy.assertText(dashboardDeals.rowIndex.exporter(), bssDeal.exporter.companyName);
 
-    cy.assertText(bankRef(bssDealId), bssDeal.bankInternalRefName);
+    cy.assertText(dashboardDeals.rowIndex.bankRef(1), bssDeal.bankInternalRefName);
```

Comment:
this is the first deal not the second deal (if its 1 indexed) the above has 2 

can these be rearranged to group together (and I dont like `dashboardDeals.rowIndex.product()` defaulting to the 1st row - its really unclear whats going on here)

Reply:
Index updated